### PR TITLE
asm.imm.str: treat shortened strings as false positives

### PR DESCRIPTION
**Pull request by kazarmy**
For #10529.

### CODE REVIEW

File: libr/util/str.c [hunk 1]
```diff
@@ -2951,7 +2951,7 @@ R_API char *r_str_from_ut64(ut64 val) {
 	if (!str) {
 		return NULL;
 	}
-	while (i < 8 && *v && *v >= '!' && *v <= '~') {
+	while (i < 8 && *v) {
```
Comment (radare) [inline libr/util/str.c]: uhm, isnt this filtering causing more false positives?

Comment (radare) [reply libr/util/str.c]: oh well you are checking this in the ds_print_ptr instead..

Comment (kazarmy) [reply libr/util/str.c]: yes

Comment (kazarmy) [reply libr/util/str.c]: `r_str_from_ut64` should just return a string and not do any filtering on it

**Pull request merged**